app/js/actions/GameActions.js

```diff
@@ -1,10 +1,10 @@
-var ApiUtils = require('../apiUtils/gameApi');
-var AppDispatcher = require('../dispatcher/appDispatcher');
-var GameConstants = require('../constants/gameConstants');
+import ApiUtils from '../apiUtils/GameApi';
+import AppDispatcher from '../dispatcher/AppDispatcher';
```

Comment:
Får feilmelding når dispatcher'n blir brukt :(
`Uncaught TypeError: _AppDispatcher2.default.dispatch is not a function`

Får titte mer på det i morgen.